src/resources/ble_esp32.cc

```diff
@@ -1057,8 +1065,26 @@ PRIMITIVE(get_characteristics_value) {
   } else {
     ALLOCATION_FAILED;
   }
+}
+
+PRIMITIVE(set_preferred_mtu) {
+  ARGS(int, mtu);
+
+  int ret_val = ble_att_set_preferred_mtu(mtu);
```

Comment:
We typically try to avoid abbreviations, but I know it is hard in this ble/mtu/att world :)

```suggestion
  int result = ble_att_set_preferred_mtu(mtu);
```